src/posit/connect/content.py

```diff
@@ -160,6 +382,10 @@ def owner(self) -> ContentItemOwner:
             )
         return ContentItemOwner(self.config, self.session, **self["owner"])
 
+    @property
+    def _variants(self) -> Variants:
+        return Variants(self.config, self.session, self.guid)
```

Comment:
This is intentionally marked as private. We don't want customers to interact directly with the variants API since it isn't a v1 API.

Reply:
We'll add a v1 version, but even so, you'll still have to fall back to the old APIs when you're on old (well, contemporary) Connect. This is fine for this PR, since I'm guessing you're not implementing all of the Variants features (I haven't read that far yet), but just wanted to put that out there for the future. It's been in production for years, people use it, so our notion of internal vs. public doesn't really matter.